Mu2eG4/inc/Mu2eG4MTRunManager.hh

```diff
@@ -61,9 +65,7 @@ namespace mu2e {
     G4VUserPhysicsList* physicsList_;
 
     int rmvlevel_;
-
-    G4int maxNumEventstoSeed_;
-
+    long initialSeed_;
```

Comment:
See previous comments about using the SeedService.

Reply:
I have removed the initialSeed FHICL parameter. We are now using the Seed Service to provide the seed to the random number generator used by the Mu2eG4 master run manager.